clients/web/src/lib/components/atoms/form/Checkbox/Checkbox.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import {slide} from "svelte/transition";
+    import {Check} from "@steeze-ui/heroicons";
+    import {Icon} from "@steeze-ui/svelte-icon";
+    import {nanoid} from "nanoid";
+    import type {FormControlState} from "../form.types";
+
+    export let label: string;
+    export let value: boolean | undefined = undefined;
+
+    export let disabled: boolean = false;
+    export let state: FormControlState = "none";
+    export let error: string | undefined = undefined;
+
+    const labelId = `checkbox_${nanoid()}`;
+    const errorId = `checkbox-error_${nanoid()}`;
+</script>
+
+<div class="state-{state}">
+    <div class="flex items-center">
+        <div class="relative w-5 h-5">
+            <input
+                id={labelId}
+                type="checkbox"
+                bind:checked={value}
+                class="appearance-none absolute inset-0 border border-gray-600 rounded focus:ring-2 focus:ring-primary/50 bg-gray-700 checked:bg-primary disabled:bg-gray-600 disabled:cursor-not-allowed disabled:border-gray-500 disabled:text-gray-400"
```

Comment:
We might need to come up with some max line length rules for these classes; this is pretty chonky

Reply:
This is the tailwind way